pkg/doc/signature/suite/bbsblssignature2020/verifier.go

```diff
@@ -0,0 +1,41 @@
+package bbsblssignature2020
+
+import (
+	"strings"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/bls"
+	sigverifier "github.com/hyperledger/aries-framework-go/pkg/doc/signature/verifier"
+)
+
+type Verifier struct {
+	bls bls.Bls
+}
+
+func NewVerifier(bls bls.Bls) *Verifier {
+	return &Verifier{
+		bls: bls,
+	}
+}
+
+// Verify will verify a signature.
+func (v *Verifier) Verify(pubKeyValue *sigverifier.PublicKey, doc, signature []byte) error {
```

Comment:
Here we verify `doc` built by the Linked Data Proof mechanism using real `bls verifier`.